go.mod

```diff
@@ -50,120 +48,118 @@ require (
 	github.com/rancher/wrangler v1.1.1
 	github.com/sirupsen/logrus v1.9.3
 	github.com/urfave/cli/v2 v2.25.7
-	golang.org/x/sync v0.5.0
-	k8s.io/api v0.27.10
-	k8s.io/apimachinery v0.27.10
-	k8s.io/client-go v0.27.10
-	k8s.io/kubernetes v1.27.10
+	golang.org/x/sync v0.7.0
+	k8s.io/api v0.30.0
+	k8s.io/apimachinery v0.30.0
+	k8s.io/client-go v0.30.0
+	k8s.io/kubernetes v1.21.0
```

Comment:
Note: those are overwritten by the replace directives.

Reply:
Why is `k8s.io/kubernetes` downgraded from `v1.27.10` to `v1.21.0`? I know we are replacing it but it seems weird that these aren't in line with the replace directives.

Reply:
One possible reason is that I ran `go mod tidy` multiple times during the developing process, and the version was changed to v1.21.0 in one run, and later I updated the replace directive so the value here was ignored and therefore to be left as v1.21.0 

Now, the version is changed to v1.30.0. running `go mod tidy` does not change the go.sum file.